server/datastore/mysql/migrations/tables/20250528102714_AddSourceColumnToHostCertificates.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250528102714, Down_20250528102714)
+}
+
+func Up_20250528102714(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+		ALTER TABLE host_certificates
```

Comment:
Altered `host_certificates` and not `host_mdm_managed_certificates` as we will present all _system_ and _login_ certificates, so we need this source for all certs, not just managed ones. (https://fleetdm.slack.com/archives/C03C41L5YEL/p1748441319246829?thread_ts=1748440394.025809&cid=C03C41L5YEL)